django/thunderstore/api/urls.py

```diff
@@ -0,0 +1,43 @@
+from django.urls import path
+
+from thunderstore.api.cyberstorm.views import (
+    CommunitiesAPIView,
+    CommunityAPIView,
+    PackageAPIView,
+    PackagePreviewAPIView,
+    TeamAPIView,
+    UserAPIView,
+)
+
+urls = [
+    path(
+        "c/",
+        CommunitiesAPIView.as_view(),
+        name="api.communities",
+    ),
+    path(
+        "c/<slug:community_identifier>/",
+        CommunityAPIView.as_view(),
+        name="api.community",
+    ),
+    path(
+        "c/<slug:community_identifier>/p/<slug:package_namespace>/<slug:package_name>/",
+        PackageAPIView.as_view(),
+        name="api.package",
+    ),
+    path(
+        "p/",
+        PackagePreviewAPIView.as_view(),
+        name="api.packages",
+    ),
+    path(
+        "t/<slug:team_identifier>/",
+        TeamAPIView.as_view(),
+        name="api.team",
+    ),
+    path(
+        "u/<slug:user_identifier>/",
+        UserAPIView.as_view(),
+        name="api.user",
+    ),
+]
```

Comment:
We don't need to use the shorthand URL structure here, nor follow the URL docs in the first place if something else makes more sense. This is the API after all, legibility to developers matters much more than short URLs.

This is a blocking issue as any future changes to the URLs would break the API.

Reply:
I'm not sure what your change request is here? Or I'm not familiar with different (longhand) url structure your referring to? Or I might be, but just connecting the dots right now, to what you are referring to.

Reply:
For example:
- `c/` -> `community/`
- `t/` -> `team/`

Normal REST convention would be to base URLs on the names of the resources (as slugs) and optionally how they relate to each other in the hierarchy (such as packages being under specific communities in order to get the community-specific package listing).

We don't follow that convention 100% of the time due to many of our resources using multiple parameters as the key (leading to nested structures), but it's the general direction we should aim for.